configs.py

```diff
@@ -16,6 +16,8 @@
 hyper_parameter_path = 'hyper_parameters.json'
 feature_lstm_ae = 'features_lstm_ae.json'
 feature_path = 'features.json'
+learning_model_path = 'learning_models.json'
+pred_data_path = 'predicted_test_data.csv'

```

Comment:
This are for a new generic style. You need to show where you put your file which has information about the model you may want to predict or train. 'pred_data_path' is the path you can save your .csv file with the prediction. make sure you run "main.py prediction 0/1 all" in order to get all models you have implemented with the argument is 'all'. Otherwise, you only get the prediction of a specific model.